src/index.html

```diff
@@ -1,15 +1,201 @@
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+                user-scalable=no,
+                initial-scale=1.0,
+                maximum-scale=1.0,minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+    >
+      <fieldset class="box">
+        <legend>
+          Personal information
+        </legend>
+
+        <label class="inside_text" for="surname">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            min="5"
+            max="10"
+          >
+        </label>
+
+        <label class="inside_text" for="name">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+          >
+        </label>
+
+        <label class="inside_text" for="age">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="18"
+            max="100"
+          >
+        </label>
+
+        <label class="inside_text" for="date_birth">
+          Full date of birth:
+          <input
+            type="date"
+            name="date_birth"
+            id="date_birth"
+          >
+        </label>
+
+        <label class="inside_text" for="terms">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            required
+            name="terms"
+            id="terms"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>
+          Registration
+        </legend>
+
+        <label class="inside_text" for="email">
+          E-mail:
+          <input
+            name="email"
+            id="email"
+            type="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="inside_text" for="password">
+          Password:
+          <input
+            name="password"
+            id="password"
+            type="password"
+            minlength="5"
+            maxlength="16"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <div class="inside_text">
+          Do you love cats?
+          <label for="love">
```

Comment:
```suggestion
          <label for="love-cats">
```
be more specific with naming